distrho/src/DistrhoUIVST3.cpp

```diff
@@ -168,8 +182,12 @@ class UIVst3
         if (fConnection != nullptr)
             connect(fConnection);
 
-#if !DISTRHO_PLUGIN_HAS_EXTERNAL_UI && (defined(DISTRHO_OS_MAC) || defined(DISTRHO_OS_WINDOWS))
+#if defined(DISTRHO_OS_MAC) || defined(DISTRHO_OS_WINDOWS)
+# if DISTRHO_PLUGIN_HAS_EXTERNAL_UI
+        nativeIdleTimerCreate(DPF_VST3_TIMER_INTERVAL);
```

Comment:
I think it makes sense to put all the idle stuff together, so move the line below `fUI.addIdleCallbackForVST3(this, DPF_VST3_TIMER_INTERVAL);` into this new function, way less confusing code that way

Reply:
Sorry didn't get this one, there is nothing below `fUI.addIdleCallbackForVST3(this, DPF_VST3_TIMER_INTERVAL);`. What should be moved into `nativeIdleTimerCreate()`?